salt/metalk8s/repo/installed.sls

```diff
@@ -18,11 +18,14 @@
 
 include:
   - .configured
+  - metalk8s.container-engine.running
 
 Inject nginx image:
   containerd.image_managed:
     - name: {{ image_fullname }}
     - archive_path: {{ archives[saltenv].path }}/images/{{ image_name }}-{{ image_version }}.tar
+    - require:
+      - test: Container engine is ready
```

Comment:
This doesn't exist if `kubelet.container_engine` is `false` (`test: No container engine to check` would).

Reply:
Hmm you right if container engine is not define I think we should safely ignore the "container engine is ready". I will require the full sls